src/amo/components/Footer.js

```diff
@@ -11,25 +11,22 @@ import './Footer.scss';
 export class FooterBase extends React.Component {
   static propTypes = {
     i18n: PropTypes.object.isRequired,
-    lang: PropTypes.string.isRequired,
   }
 
   render() {
-    const { i18n, lang } = this.props;
+    const { i18n } = this.props;
 
     return (
       <footer className="Footer">
-        <LanguagePicker lang={lang}
-          ref={(ref) => { this.languagePicker = ref; }} />
+        <LanguagePicker ref={(ref) => { this.languagePicker = ref; }} />
         <ul className="Footer-links">
           <li>
-            <Link to={`/${lang}/firefox/privacy`}
-              className="Footer-link">
+            <Link to={'/firefox/privacy'} className="Footer-link">
               {i18n.gettext('Privacy policy')}
             </Link>
           </li>
           <li>
-            <Link to={`/${lang}/firefox/legal`} className="Footer-link">
+            <Link to={'/firefox/legal'} className="Footer-link">
```

Comment:
You're right, I wanted to emulate `Link` as much as possible so I figured the leading slash should be in `to`. I'll fix it up so that's the case, but if people disagree it can be easily changed.